semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolInformationOps.scala

```diff
@@ -184,24 +183,27 @@ trait SymbolInformationOps { self: Scalacp =>
               }
               val stpe = tpe.toSemantic(linkMode)
               if (sym.isConstructor) {
-                stpe.map(_.update(_.methodType.optionalReturnType := None))
+                stpe match {
+                  case s.MethodType(stparams, sparams, _) =>
+                    s.MethodType(stparams, sparams, s.NoType)
+                  case _ => stpe
+                }
```

Comment:
Also please change to use `.copy(ret = s.NoType)` like in metac.

Reply:
Done. With `t.copy(returnType := s.NoType)` it's quite nice.